tests/cli/keyboard_interface.c

```diff
@@ -146,7 +146,7 @@ void setup_keyboard() {
         // Set bitmap for keyboard
         for (int i = 0; i < NUM_KEYBOARD_BUTTONS; i++) {
             if (keyboard_buff[i] == '1') {
-                keyboard_buttons |= (1L << i);
+                keyboard_buttons |= (1LL << i);
```

Comment:
Let's typecast the 1
I think it would be more clear and easy to understand at a glance.